pages/services.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
```

Comment:
This should be there in `components/pages/Rental/index.js`.

Reply:
My assignment is for service and repair, not rental. So should it be 
components/pages/ServicesRepair/index.js  ?

Reply:
Yeap.